### ISSUE TITLE
Picking valid choice module

### ISSUE THREAD

**Issue by AyronK** [AyronK/softmax-dnn-example#4]
App should have a component responsible for picking first valid option from NN output.

### Example for hearthstone cards:
Neural network top matches are:
- 53% `River Crocolisk`
- 47% `Chilwind Yeti`
- 37% `Boulderfist Ogre`
Player does not have `River Crocolisk` and `Chilwind Yeti` in their collection though.
In this situation component should return `Boulderfist Ogre` as the first valid match.

### TODO
- [ ] Implementation
